Formula/freedink-data.rb

```diff
@@ -0,0 +1,18 @@
+class FreedinkData < Formula
+  desc "Data for FreeDink."
```

Comment:
See e.g. [`cracklib.rb`](https://github.com/Homebrew/homebrew-core/blob/312a38050f7bf3fa4f644a9d6d23d2d9dabac506/Formula/cracklib.rb#L18). You have to declare the external resource in a `resource do` block:

```ruby
resource "freedink-data" do
  url "..."
  sha256 "..."
end
```

Then in your `def install` you can unpack it with `Resource#stage`:
```ruby
resource("freedink-data").stage do
  # execute some code here to install the additional resource
  # the freedink-data resource is unpacked in a temporary
  # directory and your code is run in that directory.
end
```